source/form.html

```diff
@@ -54,6 +54,183 @@
+                  <td class="calendar__item">15</td>
+                </tr>
+                <tr>
+                  <td class="calendar__item">16</td>
+                  <td class="calendar__item">17</td>
+                  <td class="calendar__item">18</td>
+                  <td class="calendar__item">19</td>
+                  <td class="calendar__item">20</td>
+                  <td class="calendar__item">21</td>
+                  <td class="calendar__item">22</td>
+                </tr>
+                <tr>
+                  <td class="calendar__item">23</td>
+                  <td class="calendar__item">24</td>
+                  <td class="calendar__item">25</td>
+                  <td class="calendar__item calendar__item--active  calendar__item--chosen">26</td>
+                  <td class="calendar__item calendar__item--active">27</td>
+                  <td class="calendar__item calendar__item--active">28</td>
+                  <td class="calendar__item calendar__item--available">29</td>
+                </tr>
+                <tr>
+                  <td class="calendar__item calendar__item--available">30</td>
+                  <td class="calendar__item calendar__item--available">31</td>
+                  <td class="calendar__item calendar__item--available">1</td>
+                  <td class="calendar__item calendar__item--available">2</td>
+                  <td class="calendar__item calendar__item--available">3</td>
+                  <td class="calendar__item calendar__item--available">4</td>
+                  <td class="calendar__item calendar__item--available">5</td>
+                </tr>
+              </table>
+              <a href="#" class="steps-list__button button">Следующий шаг</a>
+            </li>
+            <li class="steps-list__item step-2">
+              <h3 class="steps-list__title">Шаг 2. Маршрут</h3>
+              <p class="steps-list__desc">Укажите страны, которые вы хотели посетить. Это может быть одна или сразу несколько.</p>
+              <div class="route">
+                <input class="route__input" type="text" value="Босния и герцегов">
+                <button class="route__more" type="button"></button>
+                <span class="route__flag-wrapper"><img class="route__flag" src="https://via.placeholder.com/35x24" alt=""></span>
+                <button class="route__close" type="button" aria-label="закрыть"></button>
+              </div>
+              <div class="route">
+                <input class="route__input" type="text" value="Чехия">
+                <button class="route__more" type="button"></button>
+                <span class="route__flag-wrapper"><img class="route__flag" src="https://via.placeholder.com/35x24" alt=""></span>
+                <button class="route__close" type="button" aria-label="закрыть"></button>
+              </div>
+              <div class="route">
+                <div class="route__input">Выберите страну</div>
```

Comment:
```suggestion
                <input class="route__input" type="text" value="">
```

Надпись **Выберите страну** можно реализовать через `label`.